src/components/pools/PoolRow.tsx

```diff
@@ -257,7 +257,7 @@ export default function PoolRow({
               ? stakingInfo?.totalRewardRate?.multiply(BIG_INT_SECONDS_IN_WEEK)?.toFixed(0, { groupSeparator: ',' }) ??
               '-'
               : '0'}
-            {' ZERO / week'}
+            {` ${(stakingInfo && stakingInfo?.lpTokenName) ? stakingInfo?.lpTokenName : 'ZERO'} / week`}
```

Comment:
why `stakingInfo && stakingInfo?.lpTokenName` if you have operator `?.` 
can be replaced with `stakingInfo?.lpTokenName ?? 'ZERO'`